src/protagonist/API/Features/Image/Ingest/AssetProcessor.cs

```diff
@@ -91,7 +92,15 @@ public async Task<ProcessAssetResult> Process(AssetBeforeProcessing assetBeforeP
 
                 counts.CustomerStorage.NumberOfStoredImages++;
             }
-            else if (assetBeforeProcessing.DeliveryChannelsBeforeProcessing.IsNullOrEmpty() && alwaysReingest)
+            else if (assetBeforeProcessing.DeliveryChannelsBeforeProcessing == null && alwaysReingest)
+            {
+                assetBeforeProcessing.DeliveryChannelsBeforeProcessing = assetFromDatabase.ImageDeliveryChannels
+                    .Select(d => new DeliveryChannelsBeforeProcessing(d.Channel, d.DeliveryChannelPolicy.Name))
+                    .ToArray();
+
+                updatedDeliveryChannels = true;
+            }
+            else if (assetBeforeProcessing.DeliveryChannelsBeforeProcessing?.Length == 0 && alwaysReingest)

```

Comment:
To catch `"deliveryChannels": []` we could use fluent-validator before doing any work.